pages/common/bats.md

```diff
@@ -0,0 +1,20 @@
+# bats
+
+> Bash Automated Testing System is a TAP (https://testanything.org/) compliant testing framework for Bash.
+> More information: <https://github.com/bats-core/bats-core>.
```

Comment:
```suggestion
> More information: <https://bats-core.readthedocs.io/en/stable/usage.html>.
```
Always search for the project documentation. The priority is:
1. Official documentation
2. Official VCS (Version Control System, e.g. GitHub) link, if the README contain at least some examples
3. manned.org man page link